tests_utils/integration_fabfile.py

```diff
@@ -37,7 +37,7 @@
 # Default to all targets
 if not env.hosts:
     env.hosts = ['leonardo-00-ci', 'm3-00-ci', 'a8-00-ci', 'fox-00-ci']
-    env.hosts += ['samr21-00-ci']
+    env.hosts += ['samr21-00-ci', 'zigduino-00-ci']
```

Comment:
This shouldnt be needed anymore with the new fabric ci development on install lib,

Reply:
yes but I didn't deploy the new ci management with intsall-lib and ci.inria.fr
so it will be not needed :)